src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Startup.cs

```diff
@@ -76,6 +79,19 @@ public void ConfigureServices(IServiceCollection services)
                 .AddUrlHelper()
                 .AddHealthChecks();
 
+            if (Environment.IsDevelopment())
+            {
+                services.AddDistributedMemoryCache();
+            }
+            else
+            {
+                var config = Configuration.GetSection(ProviderCommitmentsConfigurationKeys.DataProtectionConnectionStrings).Get<DataProtectionConnectionStrings>();
+                services.AddStackExchangeRedisCache(options =>
+                {
+                    options.Configuration = config.RedisConnectionString;
```

Comment:
should apply the defaultDatabase to the config: `options.Configuration = $"{redisConnectionString},DefaultDatabase=1";`

Reply:
made the config changes